src/ex1/style.css

```diff
@@ -0,0 +1,96 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+button {
+  cursor: pointer;
+  background-color: cornflowerblue;
+  border: none;
+  color: white;
+  padding: 8px 16px;
+  margin: 8px 6px;
+  border-radius: 3px;
+  position: absolute;
+  right: 10px;
+}
+
+.background {
+  background: cadetblue;
+  min-height: 100vh;
+}
+
+.content {
+  background: white;
+  padding: 15px;
+  border-radius: 3px;
+  min-height: 500px;
+  width: 400px;
+  position: absolute;
+  top: 6%;
+  left: 50%;
+  transform: translate(-50%, 0);
```

Comment:
Much easier to achieve with flex-box and is considered a better practice in general - https://css-tricks.com/snippets/css/a-guide-to-flexbox/